src/consumer/component-ops/watch-components.ts

```diff
@@ -40,9 +34,7 @@ export default class WatchComponents {
           log(event, path);
         });
       }
-      watcher.on('ready', () => {
-        log(chalk.yellow(STARTED_WATCHING_MSG));
-      });
+      watcher.on('ready', () => {});
```

Comment:
@urigat , see my comment above. 
This is the legacy watcher. It should not be changed.

Reply:
Fixed.